Assets/Scripts/PriceManager.cs

```diff
@@ -217,6 +217,44 @@ public class PriceManager : MonoBehaviour
         return total / Math.Min(currentDay, 3.00);
     }
 
+    public static double buyProb(string nftId) {
+        Nft currentNft = nftsDict[nftId];
+
+        double total = 0;
+
+        if (futureTrendingFacePos == currentNft.imagePics[0]) {
+            total += 1;
+        }
+
+        if (futureTrendingHatPos == currentNft.imagePics[2]) {
+            total += 1;
+        }
+
+        if (futureTrendingBodyPos == currentNft.imagePics[3]) {
+            total += 1;
+        }
+        return total / Math.Min(currentDay, 3.00);
+    }
+
+    public static double sellProb(string nftId) {
+        Nft currentNft = nftsDict[nftId];
+
+        double total = 0;
+
+        if (futureTrendingFaceNeg == currentNft.imagePics[0]) {
+            total += 1;
+        }
+
+        if (futureTrendingHatNeg == currentNft.imagePics[2]) {
+            total += 1;
+        }
+
+        if (futureTrendingBodyNeg == currentNft.imagePics[3]) {
+            total += 1;
+        }
+        return total / 3.00;
```

Comment:
@prupatna Please make this the same as line 236!

Reply:
@dedeep-usc, Updated it.